cmd/notation/verify.go

```diff
@@ -161,12 +171,19 @@ func resolveReference(ctx context.Context, opts *SecureFlagOpts, reference strin
 	return ref, nil
 }
 
-func printMetadataIfPresent(outcome *notation.VerificationOutcome) {
+func printResult(outputFormat string, reference string, outcome *notation.VerificationOutcome) error {
```

Comment:
```suggestion
func printResult(outputFormat, reference string, outcome *notation.VerificationOutcome) error {
```

Reply:
huh, didn't realize it wasn't necessary, is this a go thing?